.github/workflows/ui_goerli.yml

```diff
@@ -0,0 +1,36 @@
+name: /ui (Goerli) CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ui
+
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+        cache-dependency-path: ui/yarn.lock
+    - run: npm install --global yarn
+    - run: echo $(yarn --version)
+    - run: cp .env.goerli .env.local
+    - run: yarn install
+    - run: yarn build
```

Comment:
Should it lint and then build?

Reply:
@gregfromstl Good question, I haven't thought about deeply yet.  Theoretically the application might still work for the end-user even if the linting fails.  While a build error would be more fatal, so being notified early of a build error would have higher priority than a linting error when looking at the purpose of the CI in this way.  What do you think?